doc/source/news.rst

```diff
@@ -5,6 +5,26 @@
 News
 ====
 
+.. _release-12-0-9:
+
+Release 12.0.9 - 2022-10-28
+---------------------------
+
+Improvements
+------------
+
+* [:doc:`reference/functions/escalate`] Added a document for the ``escalate()`` function.
+
```

Comment:
We need to add https://github.com/groonga/groonga/commit/0c1280c129c4e5fe7bdccad64c8b19f2c76b8e66.

Reply:
Added